validator/val_sigcrypt.c

```diff
@@ -1386,6 +1436,14 @@ check_dates(struct val_env* ve, uint32_t unow,
 		sigdate_error("verify: inception after expiration, "
 			"signature bad", expi, incep, now);
 		*reason = "signature inception after expiration";
+		if (reason_bogus){
+			/* from RFC8914 on Signature Not Yet Valid: The resolver
+			 * attempted to perform DNSSEC validation, but no
+			 * signatures are presently valid and at least some are
+			 * not yet valid. */
+			*reason_bogus = LDNS_EDE_SIGNATURE_NOT_YET_VALID;
```

Comment:
I don't know... there is also something to say for LDNS_EDE_SIGNATURE_EXPIRED because it is expired before it get's valid, but maybe we need another EDE code: SIGNATURE_EXPIRED_BEFORE_VALID. WDYT?